app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -71,21 +76,33 @@ class RecentPatientsView(context: Context, attrs: AttributeSet) : FrameLayout(co
         .ofType<RecentPatientItemClicked>()
         .takeUntil(RxView.detaches(this))
         .subscribe { openPatientSummary(it.patientUuid) }
+
+    adapterEvents
+        .ofType<SeeAllItemClicked>()
+        .takeUntil(RxView.detaches(this))
+        .subscribe { openRecentPatientsScreen() }
   }
 
   private fun screenCreates() = Observable.just(ScreenCreated())
 
   fun updateRecentPatients(recentPatients: List<RecentPatientItem>) {
-    recentPatients.forEach { it.uiEvents = adapterEvents }
-    groupAdapter.update(recentPatients)
+    val allItemTypes = recentPatients + SeeAllItem
```

Comment:
Instead of doing this here, the full set must be constructed in the `RecentPatientsViewController` so that it can be tested.

In addition, we might want to disable the `See All` button for now, and then push the code to `master` until the feature is complete. Adding this item type in the screen does not allow us to do that in a nice way.

Reply:
Right, I will include this once the feature is complete.